internal/cli/alerts/settings_update.go

```diff
@@ -38,6 +38,9 @@ func (opts *ConfigUpdateOpts) initStore() error {
 	return err
 }
 
+var updateTemplate = `Alert configuration {{.ID}} updated.
+`
```

Comment:
[non-blocking] no need for a text block if you don't need it
```suggestion
var updateTemplate = "Alert configuration '{{.ID}}' updated.\n"
```